multisig/proposals/arbitrumsepolia/vip-003/index.ts

```diff
@@ -0,0 +1,179 @@
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [XVS_BRIDGE_ADMIN_PROXY, "setPayloadSizeLimit(uint16,uint256)", arbitrumsepolia.NORMAL_TIMELOCK],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [XVS_BRIDGE_ADMIN_PROXY, "setWhitelist(address,bool)", arbitrumsepolia.NORMAL_TIMELOCK],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [XVS_BRIDGE_ADMIN_PROXY, "setConfig(uint16,uint16,uint256,bytes)", arbitrumsepolia.NORMAL_TIMELOCK],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [XVS_BRIDGE_ADMIN_PROXY, "sweepToken(address,address,uint256)", arbitrumsepolia.NORMAL_TIMELOCK],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [XVS_BRIDGE_ADMIN_PROXY, "updateSendAndCallEnabled(bool)", arbitrumsepolia.NORMAL_TIMELOCK],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [XVS, "mint(address,uint256)", XVS_BRIDGE_DEST],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [XVS, "burn(address,uint256)", XVS_BRIDGE_DEST],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [XVS_BRIDGE_ADMIN_PROXY, "setTrustedRemoteAddress(uint16,bytes)", arbitrumsepolia.NORMAL_TIMELOCK],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [XVS_BRIDGE_ADMIN_PROXY, "transferBridgeOwnership(address)", arbitrumsepolia.NORMAL_TIMELOCK],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [XVS, "migrateMinterTokens(address,address)", arbitrumsepolia.NORMAL_TIMELOCK],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [XVS, "setMintCap(address,uint256)", arbitrumsepolia.NORMAL_TIMELOCK],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [XVS, "updateBlacklist(address,bool)", arbitrumsepolia.NORMAL_TIMELOCK],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [XVS, "pause()", arbitrumsepolia.NORMAL_TIMELOCK],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [XVS, "unpause()", arbitrumsepolia.NORMAL_TIMELOCK],
+    },
+    { target: XVS_BRIDGE_ADMIN_PROXY, signature: "acceptOwnership()", params: [] },
+    {
+      target: XVS_BRIDGE_ADMIN_PROXY,
+      signature: "setTrustedRemoteAddress(uint16,bytes)",
+      params: [10102, "0x0E132cd94fd70298b747d2b4D977db8d086e5fD0"],
+    },
```

Comment:
We should enable also the bridges Arbitrum Sepolia <-> Ethereum Sepolia, and Arbitrum Sepolia <-> opBNB testnet

Reply:
I think we would need to add more commands to this multisig TX (setting trusted remote addresses, minDstGas and limits). And we would need also a multisig TX for Ethereum Sepolia and for opBNB testnet

Reply:
**Arbitrum Sepolia <-> Ethereum Sepolia, and Arbitrum Sepolia <-> opBNB testnet:** https://github.com/VenusProtocol/vips/pull/249/commits/742a0490e837fb746676274c7db2e8918fd569d7

**Ethereum Sepolia <->  Arbitrum Sepolia, and opBNB testnet  <-> Arbitrum Sepolia:** https://github.com/VenusProtocol/vips/pull/249/commits/14f4227881b2b9a034cf08560e2de6bd54e939cd